### PR TITLE
[ft #159285454] Modify add entry to use real data and add tests

### PR DESCRIPTION
**Pull request by uchemukolo**
#### What does this PR do?
- Modify add entry endpoint to use real data 
- Write test for the endpoint
#### Description of Task to be completed?
- Modify add entry endpoint to use real data 
- Write test for the endpoint
#### How should this be manually tested?
- npm run test
#### Any background context you want to provide?
- n/a
#### What are the relevant pivotal tracker stories?
- [#159285454](<https://www.pivotaltracker.com/story/show/159285454>)
#### Screenshots (if appropriate)
image

#### Questions:

### CODE REVIEW

File: server/controllers/entries.js [hunk 1]
```diff
@@ -1,133 +1,197 @@
...
+  *
+  * @return {object} return object as response
+  *
+  * @memberof Entries
+  */
+  static createEntry(request, response) {
+    const {
+      title, mood, entry
+    } = request.body;
+    console.log(request.decoded);
```
Comment (fob413) [inline server/controllers/entries.js:34]: Please remove this `console.log`

File: server/controllers/entries.js [hunk 2]
```diff
@@ -1,133 +1,197 @@
...
+      })
+      .catch((error) => {
+        response.status(500).send({
+          message: 'Create Entry Failed',
+          error: error.message,
+          status: 'fail'
+        });
+      });
   }
+// /**
```
Comment (fob413) [inline server/controllers/entries.js:58]: what's the reason for this commented code?

File: server/routes/entryRoute.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+import auth from '../helpers/authentication';
 
 const router = express.Router();
 
-router.get('/', entry.getAll);
-router.get('/:entryId', validate.entryId, entry.getOne);
-router.post('/', validate.createEntry, entry.createEntry);
-router.put('/:entryId', validate.entryId, validate.modifyEntry, entry.modifyEntry);
-router.delete('/:entryId', validate.entryId, entry.deleteEntry);
+// router.get('/', entry.getAll);
```
Comment (fob413) [inline server/routes/entryRoute.js:8]: please remove commented code if you're no longer using it

File: server/tests/entries.tests.js [hunk 4]
```diff
@@ -5,60 +5,47 @@ import app from '../app';
...
+      username: 'johndoe',
+      password: 'abcd1234'
+    };
+    chai.request(app)
+      .post('/api/v1/auth/login')
+      .send(user)
+      .end((error, response) => {
+        token = response.body.token;
+        done();
+        console.log('>>>>>>>>>>>>>>>', response.body.token);
```
Comment (fob413) [inline server/tests/entries.tests.js:24]: Please remove `console.log`

File: server/tests/entries.tests.js [hunk 5]
```diff
@@ -135,144 +121,194 @@ describe('API Integration Tests', () => {
...
             'The entry field is required.'
           ]
         };
         expect(response.status).to.equal(400);
-        expect(response.body.should.be.a('object'));
         expect(response.body).to.haveOwnProperty('message').to.eql(message);
         done();
       });
   });
+  // describe('Get All Entries', () => {
```
Comment (fob413) [inline server/tests/entries.tests.js:136]: what's the reason for this commented code

File: server/tests/users.test.js [hunk 6]
```diff
@@ -128,7 +128,7 @@ describe('MyDiary App ::: User', () => {
         expect(response.body).to.haveOwnProperty('token');
         expect(response.body).to.haveOwnProperty('newUser');
         done();
-        console.log('<<<>>>>>>>>>>>', token);
+        console.log('<<<>>>>>>>>>>>', response.body.token);
```
Comment (fob413) [inline server/tests/users.test.js:131]: Please remove all `console.log`

File: server/controllers/entries.js (hunk 2 above)
Comment (uchemukolo) [reply server/controllers/entries.js:58]: those are the endpoints i'm modifying, thats why i commented them out.

File: server/routes/entryRoute.js (hunk 3 above)
Comment (uchemukolo) [reply server/routes/entryRoute.js:8]: I will remove them

File: server/tests/entries.tests.js (hunk 4 above)
Comment (uchemukolo) [reply server/tests/entries.tests.js:24]: okay

File: server/tests/entries.tests.js (hunk 5 above)
Comment (uchemukolo) [reply server/tests/entries.tests.js:136]: those are the tests for the endpoints i'm modifying, thats why i commented them out.

File: server/tests/users.test.js (hunk 6 above)
Comment (uchemukolo) [reply server/tests/users.test.js:131]: okay

File: server/controllers/entries.js (hunk 1 above)
Comment (uchemukolo) [reply server/controllers/entries.js:34]: okay

**Pull request merged**